client/src/modules/reports/generate/stock_exit/stock_exit.config.js

```diff
@@ -0,0 +1,83 @@
+angular.module('bhima.controllers')
+  .controller('stock_exitController', InventoryFileConfigController);
+
+InventoryFileConfigController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService', 'AppCache', 'reportData', '$state',
+  'LanguageService',
+];
+
+function InventoryFileConfigController($sce, Notify, SavedReports, AppCache, reportData, $state, Languages) {
+  const vm = this;
+  const cache = new AppCache('configure_stock_exit_report');
+  const reportUrl = 'reports/stock/exit';
+
+  // default values
+  vm.includePatientExit = 1;
+  vm.previewGenerated = false;
+
+  // check cached configuration
+  checkCachedConfiguration();
+
+  vm.onSelectDepot = depot => {
+    vm.depot = depot;
+  };
+
+  vm.clear = key => {
+    delete vm[key];
+  };
+
+  vm.clearPreview = () => {
+    vm.previewGenerated = false;
+    vm.previewResult = null;
+  };
+
+  vm.preview = form => {
+    if (form.$invalid) { return 0; }
```

Comment:
You might want to put custom logic here to make sure that at least one checkbox is checked.  Something like:
```js
const sum = vm.includePatientExit + vm.includeServiceExit + vm.includeDepotExit + vm.includeLossExit;
const hasOneChecked = sum > 0;
if (!hasOneChecked) {
  return 0;
}
```

You might also set a message that says at least one of the options should be checked.